_notebooks/nb-stripout.py

```diff
@@ -10,8 +10,12 @@
 output_stream = io.TextIOWrapper(sys.stdout.buffer, encoding="utf-8")
 
 parser = argparse.ArgumentParser()
-parser.add_argument("-t", "--textconv", action="store_true", help="Print results to output")
-parser.add_argument("-d", "--doc-mode", action="store_true", help="fastai docs nb-specific strip out")
+parser.add_argument(
```

Comment:
Please `black *py -l 120`? 

We both use wider displays.

Reply:
Done 889e390